src/tmt300rs/hid-tmt300rs.c

```diff
@@ -370,6 +361,63 @@ static void t300rs_calculate_periodic_values(struct ff_effect *effect)
 	periodic->offset = clamp(periodic->offset, -headroom, headroom);
 }
 
+static uint16_t t300rs_condition_max_saturation(uint16_t effect_type)
+{
+	if(effect_type == FF_SPRING)
+		return 0x6AA6;
```

Comment:
Style nitpick, please stick to small hex letters.

Reply:
I addressed it in the last commit